client/assets/stylesheets/shared/functions/_z-index.scss

```diff
@@ -165,6 +165,7 @@ $z-layers: (
 		".popover.info-tooltip__container": 100300,
 		".popover.info-popover__tooltip.plugin-action__disabled-info": 100300,
 		"body .webui-popover": 100300,
+		".commands-commands-menu__overlay": 100300,
 		".fullscreen-fader": 200000,
 		".guided-tours__overlay": 200050,
 		".guided-tours__step": 201000,
```

Comment:
Out of interest, why not above the Guided Tours? The Command Palette is still the most recent action, so presumably should still be top of the stack? 

<img width="827" alt="Screenshot 2024-03-16 at 11 14 07" src="https://github.com/Automattic/wp-calypso/assets/43215253/6da7f3e2-cb4c-4cc0-961f-77abae8860ae">

Reply:
Thanks @Aurorum. What was going through my head was "what if we want to do a guided tour for the command palette". Looking at your screenshot that was clearly a bad idea. I've changed the z-index and we can tackle that problem if it ever becomes one

Reply:
Agreed, looks nicer now I think!

<img width="565" alt="Screenshot 2024-03-18 at 08 59 14" src="https://github.com/Automattic/wp-calypso/assets/43215253/2718f435-608c-4180-88a0-f68f9bd808b2">